main.py

```diff
@@ -78,27 +88,29 @@ def get_stock_data(ticker):
 		history_data = stock_data.history(period="1m")
 		if not history_data.empty and dividend_data is not None and not dividend_data.empty:
 			return trim_me(float(history_data.Close.iloc[0])), float(dividend_data.iloc[-1])
-		elif not history_data.empty and (dividend_data is None or dividend_data.empty):
```

Comment:
This can remain as an 'elif', contain only extra condition regarding 'dividend_data_empty'. It will ease the process flow. Short explanation here https://stackoverflow.com/a/9271755\
The if/elif flow doesn't need to end with 'else' if you don't need it